escrow/validation_test.go

```diff
@@ -37,12 +35,22 @@ func SignFreeTestPayment(payment *FreeCallPayment, privateKey *ecdsa.PrivateKey)
 		[]byte(payment.UserId),
 		[]byte(config.GetString(config.OrganizationId)),
 		[]byte(config.GetString(config.ServiceId)),
+		[]byte(payment.GroupId),
 		bigIntToBytes(payment.CurrentBlockNumber),
+		payment.AuthToken,
 	}, nil)
 
 	payment.Signature = getSignature(message, privateKey)
 }
 
+func GenerateFreeCallTokenPayment(payment *FreeCallPayment, privateKey *ecdsa.PrivateKey,userAddress common.Address) {
```

Comment:
Just to confirm am I right that AuthToken includes only:
- user id
- user address
- token expiry block
?
It doesn't include orgId, serviceId and groupId?

Reply:
this is correct, but when you make a call to state_service / actuall call to Daemon , you need to sign using orgid, serviceid and groupid , ( the other signture) . .. do you recommend adding this on token too?

Reply:
> do you recommend adding this on token too?

Not at all. It depends on requirements should user get new token for each service or can use one for all of them. My opinion the last will be simpler for user and for implementation so lets leave it as is.